docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -73,26 +81,26 @@ Name | Type | Description | Default | Required
 
 ### `http_client_config` block
 
-Name | Description | Required
----- | ----------- | --------
-`basic_auth`               | `basic_auth` block    | Setup of Basic HTTP authentication credentials. | | no
-`authorization`            | `authorization` block | Setup of HTTP Authorization credentials. | | no
-`oauth2`                   | `oauth2` block        | Setup of the OAuth2 client. | | no
-`tls_config`               | `tls_config` block    | Configuration options for TLS connections. | | no
-`bearer_token`             | `secret`   | Used to set up the Bearer Token. | | no
-`bearer_token_file`        | `string`   | Used to set up the Bearer Token file. | | no
-`proxy_url`                | `string`   | Used to set up a Proxy URL. | | no
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`basic_auth`               | `basic_auth` block    | Set up basic HTTP authentication credentials. | | no
+`authorization`            | `authorization` block | Set up HTTP authorization credentials. | | no
+`oauth2`                   | `oauth2` block        | Set up the OAuth2 client. | | no
+`tls_config`               | `tls_config` block    | Configure options for TLS connections. | | no
+`bearer_token`             | `secret`   | Use to set up the Bearer Token. | | no
+`bearer_token_file`        | `string`   | Use to set up the Bearer Token file. | | no
+`proxy_url`                | `string`   | Use to set up a proxy URL. | | no
 `follow_redirects`         | `bool`     | Whether the scraper should follow redirects. | `true` | no
 `enable_http_2`            | `bool`     | Whether the scraper should use HTTP2. | `true` | no
 
 The following subblocks are supported:
 
 Name | Description | Required
 ---- | ----------- | --------
-[`basic_auth`](#basic_auth-block) | Configures basic_auth for authenticating against targets | no
-[`authorization`](#authorization-block) | Configures generic authorization against targets | no
-[`oauth2`](#oauth2-block) | Configures OAuth2 for authenticating against targets | no
-[`tls_config`](#tls_config-block) | Configures TLS settings for connecting to targets | no
+[`basic_auth`](#basic_auth-block) | Configure basic_auth for authenticating against targets | no
+[`authorization`](#authorization-block) | Configure generic authorization against targets | no
+[`oauth2`](#oauth2-block) | Configure OAuth2 for authenticating against targets | no
+[`tls_config`](#tls_config-block) | Configure TLS settings for connecting to targets | no
```

Comment:
Is this right? Is this a subblock of _both_ the component and of `http_client_config` as listed above?

Reply:
The `tls_config` block can be a subblock of both `http_client_config` _and_ `http_client_config -> oauth2`, but not the component itself.

Also, I realized we may want everything in backticks here as well

Reply:
Sorry, I was commenting on the whole section here, not just `tls_config`.

I was thrown off that the arguments list for the `http_client_config` block included the name of the subblocks. I think we should drop them from the argument list and just keep them in the separate table here.